apple_sign_in.gemspec

```diff
@@ -1,34 +1,41 @@
 # frozen_string_literal: true
 
-require_relative 'lib/apple_sign_in/version'
+require_relative 'lib/apple_sign_in/base/version'
 
 Gem::Specification.new do |spec|
   spec.name          = 'apple_sign_in'
-  spec.version       = AppleSignIn::VERSION
-  spec.authors       = ['fedeagripa']
-  spec.email         = ['federicogagripa@gmail.com']
+  spec.version       = AppleSignIn::Base::VERSION
+  spec.authors       = ['Timothy Peraza']
+  spec.email         = ['timothy@rootstrap.com']
 
-  spec.summary       = ''
-  spec.description   = ''
-  spec.homepage      = ''
+  spec.summary       = 'A gem that implement an integration with apple sign in'
+  spec.homepage      = 'https://github.com/rootstrap/apple_sign_in'
   spec.license       = 'MIT'
   spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
 
-  # spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
-
-  # spec.metadata["homepage_uri"] = spec.homepage
-  # spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
-  # spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
+  spec.metadata['homepage_uri'] = spec.homepage
+  spec.metadata['source_code_uri'] = 'https://github.com/rootstrap/apple_sign_in'
+  spec.metadata['changelog_uri'] = 'https://github.com/rootstrap/apple_sign_in'
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files = Dir['LICENSE.txt', 'README.md', 'lib/**/*']
+  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
+    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  end
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'reek'
-  spec.add_development_dependency 'rspec'
-  spec.add_development_dependency 'rubocop'
+
+  # Production dependencies
+  spec.add_dependency 'jwt', '~> 2.2'
+
+  # Development dependencies
+  spec.add_development_dependency 'byebug', '~> 11.1'
+  spec.add_development_dependency 'railties', '~> 6.0'
```

Comment:
Why do we need this?

Reply:
we needed it for this : `require 'active_support/core_ext/hash'`